config/config.go

```diff
@@ -410,7 +410,7 @@ func (c *Config) UnParsedOptions() map[string]string {
 func (c *Config) dotdDir() string {
 	if !forceDotParse {
 		home, err := iu.HomeDir()
-		if err == nil {
+		if err == nil && home != "/" {
```

Comment:
This seems like quite a big special case here that just doesnt seem like it belong.  Should we fix FreeBSD rc scripts instead @smortex ?